cmd/keytransparency-server/main.go

```diff
@@ -40,6 +40,7 @@ import (
 
 	gpb "github.com/google/keytransparency/core/proto/keytransparency_v1_grpc"
 	gauth "github.com/google/keytransparency/impl/google/authentication"
+	_ "github.com/google/trillian/crypto/keys/der/proto"
```

Comment:
anony ordering again (unless goimports disagrees, in which case let it do it's thing)